ragtime.jdbc/src/ragtime/jdbc.clj

```diff
@@ -33,7 +41,6 @@
     (sql/insert! db-spec migrations-table
                  {:id         (str id)
                   :created_at (format-datetime (Date.))}))
-  
```

Comment:
That diff, yes - but if you check out that line on master it's a newline padded with two spaces.